prep-javascript-conditionals/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Conditionals</title>
+</head>
+
+<body>
+  <script>
+    // Your code here
+    var age = 29;
+
+    if (age < 18) {
+      console.log('A child');
+    } else {
+      console.log('An Adult');
+    }
+
+    var student1Score = 70;
+
+    if (student1Score >= 70) {
+      console.log('Congrats you received a passing grade!');
+    } else {
+      console.log('Oh no you failed!');
+    }
+
+    var student2Score = 100;
+
+    if (student2Score < 60) {
+      console.log('F');
+    } else if (student2Score < 70) {
+      console.log('D');
+    } else if (student2Score < 80) {
+      console.log('C');
+    } else if (student2Score < 90) {
+      console.log('B');
+    } else if (student2Score < 100) {
+      console.log('A');
+    } else {
+      console.log('A++');
+    }
+
+    var season = 'spring';
+
+    if (season === 'summer') {
+      console.log('it\'s hot today');
+    } else if (season === 'spring') {
+      console.log('the flowers are blooming');
+    } else if (season === 'autumn') {
+      console.log('the leaves are changing colors');
+    } else if (season === 'winter') {
+      console.log('it\'s cold today');
+    } else {
+      console.log('please enter a valid season');
+    }
+
+    var dayOfTheWeek = 'thursday';
+
+if (dayOfTheWeek === 'saturday' || dayOfTheWeek === 'sunday') {
+  console.log('have a good weekend!');
+} else {
+  console.log('Its a weekday!');
+}
```

Comment:
Be mindful of your indenting, you want to keep it consistent throughout the file